app/screens/user_profile/user_profile.tsx

```diff
@@ -46,11 +50,20 @@ const TITLE_HEIGHT = 118;
 const OPTIONS_HEIGHT = 82;
 const SINGLE_OPTION_HEIGHT = 68;
 const LABEL_HEIGHT = 58;
+const EXTRA_HEIGHT = 60;
+const MANAGE_ICON_HEIGHT = 72;
+
+const messages = defineMessages({
```

Comment:
`defineMessages`, why do we need it?

Reply:
Personal preferencde so that line 97 can be a one-liner